frontend/src/services/StorageService.ts

```diff
@@ -169,6 +165,8 @@ async function uploadImage(
   const fileS3Key = uuidv4().concat(extension);
   const dir = directory ? directory + "/" : "";
 
+  console.log(dir.concat(fileS3Key));
+  console.log(alternativeBucket);
```

Comment:
Did you mean to leave these console logs?

Reply:
thought I took those out. removed!